server/src/main/java/org/opensearch/telemetry/tracing/AttributeNames.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing;
+
+/**
+ * Hold the Attribute names to avoid the duplication and consistensy.
+ */
+public class AttributeNames {
```

Comment:
Please sync up with `SemanticAttributes`, most of them are there

Reply:
For my understanding, The suggestion here is to sync the attribute names from the SemanticAttributes class? Because we can not use them directly as we can't take dependency on the otel code outside of plugin.